dams-key-server/src/database/log.rs

```diff
@@ -0,0 +1,27 @@
+//! Module for operations on log entries in the database.
+//!
+//! Functions in this module are used to perform CRUD operations
+//! on the [`LogEntry`] model in the MongoDB database.
+
+use crate::{constants, DamsServerError};
+use dams::{
+    audit_log::{LogEntry, Outcome},
+    crypto::KeyId,
+    user::LogIdentifier,
+    ClientAction,
+};
+use mongodb::Database;
+
+/// Create a new [`LogEntry`] for the given actor, action, and outcome
+pub async fn create_log_entry(
+    db: &Database,
+    actor: &LogIdentifier,
```

Comment:
```suggestion
    actor: impl Into<LogIdentifier>,
```

This would let us pass user ids and account id's directly.